rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/OperatorPolicyReconciler.java

```diff
@@ -0,0 +1,81 @@
+package com.indeed.operators.rabbitmq.reconciliation.rabbitmq;
+
+import com.indeed.operators.rabbitmq.api.RabbitManagementApiProvider;
+import com.indeed.operators.rabbitmq.model.crd.rabbitmq.OperatorPolicySpec;
+import com.indeed.operators.rabbitmq.model.rabbitmq.RabbitMQCluster;
+import com.indeed.operators.rabbitmq.model.rabbitmq.RabbitMQConnectionInfo;
+import com.indeed.operators.rabbitmq.resources.RabbitMQServices;
+import com.indeed.rabbitmq.admin.RabbitManagementApi;
+import com.indeed.rabbitmq.admin.pojo.OperatorPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class OperatorPolicyReconciler {
+
+    private static final Logger log = LoggerFactory.getLogger(PolicyReconciler.class);
+
+    private final RabbitManagementApiProvider apiProvider;
+
+    public OperatorPolicyReconciler(
+            final RabbitManagementApiProvider apiProvider
+    ) {
+        this.apiProvider = apiProvider;
+    }
+
+    public void reconcile(final RabbitMQCluster cluster) {
+        final RabbitMQConnectionInfo connectionInfo = new RabbitMQConnectionInfo(cluster.getName(), cluster.getNamespace(), RabbitMQServices.getDiscoveryServiceName(cluster.getName()));
+        final RabbitManagementApi apiClient = apiProvider.getApi(connectionInfo);
+
+        deleteObsoleteOperatorPolicies(cluster, apiClient);
+
+        for (final OperatorPolicySpec desiredPolicy : cluster.getOperatorPolicies()) {
+            final OperatorPolicy policy = new OperatorPolicy()
```

Comment:
For consistency, would you want to have OperatorPolicySpec#asOperatorPolicy?  It seems like we do that everywhere else.

Reply:
hmmm yea...we do have `asDefinition` and `asOperatoryPolicyDefinition` so it might make sense here. on the other hand, i was hoping to eventually remove the rabbitmq-admin dependency from the rabbitmq-operator-model module.

i'll give it some thought

Reply:
Its not anything I'd consider a blocker.  I don't think it makes the code any more or less hard to read.